com.yubico.YubiOath.json

```diff
@@ -0,0 +1,257 @@
+		{
+                    "type": "archive",
+                    "url": "https://github.com/LudovicRousseau/PCSC/archive/pcsc-1.8.24.tar.gz",
+		    "sha256": "6b079ff4fbcbc1a181aa66b8f184a07865e1cb55d8d9b8aaa26b595bb0088f51"
+		}
+	    ]
+	},
+	{
+	    "name": "pyotherside",
+            "buildsystem": "qmake",
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "https://github.com/thp/pyotherside/archive/1.5.4.tar.gz",
+		    "sha256": "5f218778fc60377cee44b0edf6deaa6dd6aa84a109ebbb5b13d52cec31696b80"
+		},
+		{
+		    "type": "patch",
+		    "path": "pyotherside-install-path.patch"
+		}
+	    ]
+	},
+	{
+	    "name": "swig",
+	    "config-opts": [
+		"--without-alllang",
+		"--with-python3"
+	    ],
+	    "cleanup": [
+		"/**"
+	    ],
+	    "sources": [
+		{
+                    "type": "archive",
+                    "url": "http://download.sourceforge.net/swig/swig-3.0.12.tar.gz",
+		    "sha256": "7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d"
+		}
+	    ]
+	},
+	{
+            "name": "pyscard",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/LudovicRousseau/pyscard/archive/release-1.9.7.tar.gz",
+		    "sha256": "402956d4c2c0ae265ff5d590efdf80297b07060fa923230658e682cff29a27a2"
+		},
+		{
+		    "type": "patch",
+		    "path": "pyscard-include-path.patch"
+		}
+	    ]
+	},
+	{
+            "name": "pyusb",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pyusb/pyusb/archive/v1.0.2.tar.gz",
+		    "sha256": "49dfa601f28e9399df62004170cfb623197ad32d8399885b7f789775767b50c3"
+		}
+	    ]
+	},
+	{
+            "name": "click",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/pallets/click/archive/7.0.tar.gz",
+		    "sha256": "7cd752d4b5621bddf6428ca00ef5c2cd9b4b77bc41076a69eff51cbbfd4f8646"
+		}
+	    ]
+	},
+	{
+            "name": "asn1crypto",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://github.com/wbond/asn1crypto/archive/0.24.0.tar.gz",
+		    "sha256": "d2a82bb3c29e1abcb376b59679510b58dd796c813f8983f2db9b45e9ab228190"
+		}
+	    ]
+	},
+	{
+            "name": "cffi",
+            "buildsystem": "simple",
+	    "build-commands": [
+		"pip3 install --prefix /app --no-deps ."
+	    ],
+	    "sources": [
+		{
+		    "type": "archive",
+		    "url": "https://files.pythonhosted.org/packages/64/7c/27367b38e6cc3e1f49f193deb761fe75cda9f95da37b67b422e62281fcac/cffi-1.12.2.tar.gz",
```

Comment:
latest is [1.12.3](https://bitbucket.org/cffi/cffi/get/v1.12.3.tar.bz2)

Reply:
Fixed